cmd/chaos-builder/impl.go

```diff
@@ -26,12 +26,17 @@ func (in *{{.Type}}) IsDeleted() bool {
 	return !in.DeletionTimestamp.IsZero()
 }
 
-// IsPaused returns whether this resource has been paused
-func (in *{{.Type}}) IsPaused() bool {
-	if in.Annotations == nil || in.Annotations[PauseAnnotationKey] != "true" {
-		return false
+// GetPause returns the annotation when the chaos needs to be paused
```

Comment:
How to cancel pause action?

Reply:
Set `in.Annotations[PauseAnnotationKey] = ""`, as `SetPause` do. 
The current implementation is a bit counterintuitive.. Added a line of comment for the explanation. Thx~